canvas_modules/harness/README.md

```diff
@@ -19,7 +19,7 @@ limitations under the License.
 # Test Harness
 
 ## Environment setup
-- Nodejs `node 8.16.x`
+- Nodejs `node 12.16.1`
```

Comment:
We probably can just put node 12 instead of a specific version.  Same for the other places we specify node 12.16.1 specifically.

Reply:
Hi @matthoward366, I updated all references to 12.x